pkg/registry/helm_crd.go

```diff
@@ -0,0 +1,61 @@
+package registry
```

Comment:
You'd know better than me, but using an underscore feels weird here. It's minor, but it separates helm.go from helm_tests.go because this and its test are in between. From what I could find [1], this doesn't seem like it passes their exception cases for underscores. 

It's not something I'd block the PR on, I'm mostly curious to see your rationale.

[1] https://google.github.io/styleguide/go/decisions.html

Reply:
It's in fact fairly common to use underscores in filenames for compound words. I'm pretty sure I've seen such usages in all Go projects which I have followed / contributed to.

It's the package names where an underscore is definitely not appropriate. Filenames are generally fine (or at least in my opinion).

Reply:
We also already have it in EIB e.g. `pkg/kubernetes/artefact_downloader.go`.